### ISSUE TITLE
ExitFailure 1 while installing the package.

### ISSUE THREAD

**Issue by PiotrJustyna** [haskell-compat/base-orphans#26]
Hey,

I'm having some problems installing this package, would you be able to help? I couldn't find "Console.Handler" on Hackage - is this even the problem? This is one of my first cabal packages, so apologies if I'm doing something silly.

```
cabal install base-orphans -v

...

src\Data\Orphans.hs:455:29:
    Not in scope: type constructor or class `Console.Handler'
Failed to install base-orphans-0.4.0
World file is already up to date.
cabal: Error: some packages failed to install:
base-orphans-0.4.0 failed during the building phase. The exception was:
ExitFailure 1
```

**Comment by RyanGlScott** [comment]
What operating system and compiler version are you using?

**Comment by PiotrJustyna** [comment]
@RyanGlScott Thanks for replying so quickly :+1:

OS: Windows 8.1 Pro x64
Compiler: GHC 7.6.3

**Comment by RyanGlScott** [comment]
OK. I suspect this is probably Windows-related, so I'll take a look later tonight when I can use my Windows machine.

**Comment by RyanGlScott** [comment]
Yep, me being sloppy with Windows-related imports was the culprit. This should be fixed as of 5f54d0a695af46fc46f83ee9ef4c244ecc91ea13 (released on Hackage as [0.4.1](http://hackage.haskell.org/package/base-orphans-0.4.1))—let me know if you have any other issues.

**Comment by PiotrJustyna** [comment]
Thanks for addressing it so quickly. The issue is solved now.
